%YAML:1.0
---
calibration_time: "Fri 04 Jan 2019 05:53:54 PM +0330"
image_width: 640
image_height: 480
flags: 0
camera_matrix: !!opencv-matrix
   rows: 3
   cols: 3
   dt: d
   data: [ 904.041947, 0., 480.879667, 0.,
       906.260575, 343.293266, 0., 0., 1. ]
distortion_coefficients: !!opencv-matrix
   rows: 1
   cols: 5
   dt: d
   data: [ -0.052353, 0.079245,
       -0.003635, -0.001079,
       0.000000 ]
avg_reprojection_error: 7.1773643585995239e-01
